ocs_ci/ocs/pgsql.py

```diff
@@ -216,14 +251,20 @@ def wait_for_pgbench_status(self, status, timeout=None):
             timeout (int): Time in seconds to wait
 
         """
-        timeout = timeout if timeout else 1200
+        timeout = timeout if timeout else 1500
```

Comment:
Test is passing with 25mins timeout, suggested change it to change it to 30mins and keep track of time taken by each pg bench pods for futher analysis.